src/main/java/coupon/domain/vo/IssuePeriod.java

```diff
@@ -31,4 +32,12 @@ private void validate(LocalDateTime issueStartedAt, LocalDateTime issueEndedAt)
             throw new GlobalCustomException(ErrorMessage.INVALID_ISSUE_PERIOD);
         }
     }
+
+    public boolean isInIssuePeriod(LocalDateTime issueDateTime) {
```

Comment:
```suggestion
    public boolean isBetween(LocalDateTime issueDateTime) {
```
미아가 이미 도메인, 파라미터 이름을 직관적으로 잘 지어두어서 메서드 명을 더 간단하게 바꾸어도 잘 이해가 될 것 같아요~!

Reply:
그렇네요! `issuePeriod.includes(issueDateTime)`도 자연스러워 보여서 `includes`로 변경해보았습니다! 👍👍